patch from kde cvs - posted on -packager, has security significance
--- kio/kio/job.cpp  #1.427:1.428
+++ kio/kio/job.cpp  #1.427:1.428
@@ -3261,8 +3261,8 @@ void CopyJob::copyNextFile()
                            KSimpleConfig config( path );
                            config.setDesktopGroup();
-                           config.writePathEntry( QString::fromLatin1("URL"), (*it).uSource.url() );
-                           KURL urlName = (*it).uSource;
-                           urlName.setPass( "" );
-                           config.writeEntry( QString::fromLatin1("Name"), urlName.url() );
+                           KURL url = (*it).uSource;
+                           url.setPass( "" );
+                           config.writePathEntry( QString::fromLatin1("URL"), url.url() );
+                           config.writeEntry( QString::fromLatin1("Name"), url.url() );
                            config.writeEntry( QString::fromLatin1("Type"), QString::fromLatin1("Link")
 ); QString protocol = (*it).uSource.protocol();
